exercicios/para-casa/entregas/veronica-paranhos/projeto/ContaStandard/ContaStandard.js

```diff
@@ -0,0 +1,44 @@
+const Conta = require("../Conta/Conta");
+
+class ContaStandard extends Conta {
+  renda;
+  #limiteTransacional;
+
+  constructor(agencia, conta, saldo, renda) {
+    super(agencia, conta, saldo);
+    this.renda = renda;
+    this.#limiteTransacional = 1000;
+  }
+
+  criarConta(agencia, conta, saldo, renda) {
+    if (renda > 4999.99)
+      throw new Error("Renda incompatível com a criação da conta Standard.");
+
+    super.criarConta(agencia, conta, saldo);
+    return "Conta criada com sucesso!";
+  }
+
+  sacar(valor) {
+    if (valor > this.#limiteTransacional)
+      throw new Error("Limite de transações diárias atingido.");
+
+    super.sacar(valor);
+  }
```

Comment:
Sugestão: transformar condicionais mais simples em operador ternário:
`(valor <= this.#limiteTransacional) ? super.sacar(valor) : throw new Error("Limite de transações diárias atingido.");`

Reply:
Eu tbm retirei esse aqui.